src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift

```diff
@@ -258,18 +258,13 @@ class ENAUITests_01b_Statistics: CWATestCase {
 	
 	func test_screenshot_local_statistics_card() throws {
 		// GIVEN
-		let incidenceTitle = AccessibilityIdentifiers.Statistics.Combined7DayIncidence.title
 		let addStatisticsButtonTitle = AccessibilityIdentifiers.LocalStatistics.addLocalIncidencesButton
 		let localStatisticsTitle = AccessibilityIdentifiers.LocalStatistics.localStatisticsCardTitle
 
 		// WHEN
 		app.setPreferredContentSizeCategory(accessibility: .normal, size: .S)
 		app.launch()
 		app.swipeUp(velocity: .slow)
-
-
-		XCTAssert(self.app.staticTexts[incidenceTitle].waitForExistence(timeout: .medium))
```

Comment:
This is not really not needed, the screen is already initialised to what we need. It is checking for something which is only on 10-20% on the screen.